examples/example_1d.py

```diff
@@ -0,0 +1,38 @@
+from src.bayesian import run_one_1d_bayesian_optimization
+
```

Comment:
Since this is an example file, I think it would be worth having a comment at the top describing what this file does, and why it exists. Maybe 5 lines or so.

Reply:
Something like:

"This file runs the `run_one_1d_bayesian_optimization` function for example data, and then plots the output.

The first half of this file runs the `run_one_1d_bayesian_optimization` function with an example set of measurements at points. The function then returns a grid of predicted mean/variance/etc.

The second half of this file then plots the predicted mean & variance.

Reply:
I do think the file itself is well laid out, very straighforward and clear.